www/src/models/transformation/TransformationResult.ts

```diff
@@ -0,0 +1,8 @@
+import { Record } from "gena-app";
+
+export interface TransformationResult extends Record<number> {
```

Comment:
nitpicking -- transformationResult does not have id so it should not extend Record<number> ?

Reply:
updated